<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Kefa</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MLP</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(239, 239, 239);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Paint" name="Painter" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  white;
border: 1px solid gray;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>0</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  white;
border: 1px solid gray;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="setup">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>10</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">

QPushButton {
border-radius: 25px;
background-color:rgb(248, 248, 248);

border: 1px solid gray;
}

QPushButton:pressed {

border:2px solid black;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/gear_c.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;</string>
    </property>
    <property name="text">
     <string>Neural Network</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;</string>
    </property>
    <property name="text">
     <string>Type</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="graph">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>460</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>23</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton {
color: black;
background-color: rgb(239, 239, 239);
}

QRadioButton::indicator {
    width:                  10px;
    height:                 10px;
    border-radius:         5px;
}

QRadioButton::indicator:checked {
    background-color:       rgb(148, 179, 193);
    border:                 1.5px solid black;
}

QRadioButton::indicator:unchecked {
    background-color:       white;
    border:                 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string> Graph</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="matrix">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>500</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>23</pointsize>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton {
color: black;
background-color: rgb(239, 239, 239);
}

QRadioButton::indicator {
    width:                  10px;
    height:                 10px;
    border-radius:         5px;
}

QRadioButton::indicator:checked {
    background-color:       rgb(148, 179, 193);
    border:                 1.5px solid black;
}

QRadioButton::indicator:unchecked {
    background-color:       white;
    border:                 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string> Matrix</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;</string>
    </property>
    <property name="text">
     <string>Hidden Layer</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>590</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;</string>
    </property>
    <property name="text">
     <string>Weight</string>
    </property>
   </widget>
   <widget class="QPushButton" name="load_weight">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>600</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>23</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border: 1px solid gray;
background-color: white;
color: black;
}

QPushButton:pressed {
    background-color: rgb(228, 226, 229);
}</string>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>650</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;</string>
    </property>
    <property name="text">
     <string>File name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="name_weights">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>657</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;</string>
    </property>
    <property name="text">
     <string>2_layers.weights</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>410</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;</string>
    </property>
    <property name="text">
     <string>Training</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>550</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;</string>
    </property>
    <property name="text">
     <string>Testing</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="validation">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>460</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border: 1px solid gray;
background-color: rgb(189, 193, 195);
color: black;
border-radius: 10px;
}

QPushButton:pressed {
    background-color: rgb(228, 226, 229);
}</string>
    </property>
    <property name="text">
     <string>Cross-validation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="train">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>510</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border: 1px solid gray;
background-color: rgb(189, 193, 195);
color: black;
border-radius: 10px;
}

QPushButton:pressed {
    background-color: rgb(228, 226, 229);
}</string>
    </property>
    <property name="text">
     <string>Epochal training</string>
    </property>
   </widget>
   <widget class="QPushButton" name="load_image">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>600</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid gray;
	background-color: rgb(148, 179, 193);
	color: black;
	border-radius: 10px;
}

QPushButton:pressed {
    background-color: rgb(166, 204, 221);

}</string>
    </property>
    <property name="text">
     <string>Load Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="load_test">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>650</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid gray;
	background-color: rgb(148, 179, 193);
	color: black;
	border-radius: 10px;
}

QPushButton:pressed {
   background-color: rgb(166, 204, 221);

}</string>
    </property>
    <property name="text">
     <string>Load test</string>
    </property>
   </widget>
   <widget class="QLabel" name="main_letter">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>281</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Chalkboard</family>
      <pointsize>280</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  white;
color: black;</string>
    </property>
    <property name="text">
     <string>W</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="letter_1">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>320</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Chalkboard</family>
      <pointsize>57</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  white;
color: rgb(100, 100, 100);</string>
    </property>
    <property name="text">
     <string>A</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="letter_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>320</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Chalkboard</family>
      <pointsize>57</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  white;
color: rgb(121, 120, 121);</string>
    </property>
    <property name="text">
     <string>S</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="letter_3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>320</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Chalkboard</family>
      <pointsize>60</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  white;
color: rgb(150, 149, 150);</string>
    </property>
    <property name="text">
     <string>P</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="hidden_layer">
    <property name="geometry">
     <rect>
      <x>302</x>
      <y>540</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
border: 1px solid gray;

background:white;
font-size: 20px;
color: black;
selection-background-color: rgb(148, 179, 193);
}

QComboBox:on
{
selection-background-color: rgb(189, 193, 195);
}
</string>
    </property>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>400</y>
      <width>1</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>399</y>
      <width>400</width>
      <height>1</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Paint</class>
   <extends>QWidget</extends>
   <header>view/paint.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
